presto-main/src/test/java/com/facebook/presto/server/TestHttpRequestSessionContext.java

```diff
@@ -99,7 +100,7 @@ public void testPreparedStatementsHeaderDoesNotParse()
                         .put(PRESTO_PREPARED_STATEMENT, "query1=abcdefg")
                         .build(),
                 "testRemote");
-        new HttpRequestSessionContext(request);
+        new HttpRequestSessionContext(request, new SqlParserOptions());
```

Comment:
can you  also add a test that uses some special parser options and that allows the prepared statement to parse successfully?